pkg/apicapi/apicapi.go

```diff
@@ -305,6 +313,7 @@ func (conn *ApicConnection) handleQueuedDn(dn string) bool {
 		}
 	}
 
+	conn.log.Debug("<== respClasses::hasPendingChange::hasDesiredState==>:", respClasses, hasPendingChange, hasDesiredState)
```

Comment:
nit: did you intend to add this log message to this commit? Asking because the log message string format is different from other log messages.